gc/base/standard/Scavenger.cpp

```diff
@@ -4632,15 +4632,13 @@ MM_Scavenger::internalGarbageCollect(MM_EnvironmentBase *envBase, MM_MemorySubSp
 		return true;
 	}
 
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	if (IS_CONCURRENT_ENABLED && isBackOutFlagRaised()) {
 		bool result = percolateGarbageCollect(env, subSpace, NULL, ABORTED_SCAVENGE, J9MMCONSTANT_IMPLICIT_GC_PERCOLATE_ABORTED_SCAVENGE);
 
 		Assert_MM_true(result);
 
 		return true;
 	}
-#endif
```

Comment:
Why we need this change? If `OMR_GC_CONCURRENT_SCAVENGER` is not defined `IS_CONCURRENT_ENABLED` is always `false`

Reply:
we don't have to remove these ifdefs, but there is no harm to removing them. 
they were there just because J9MMCONSTANT_IMPLICIT_GC_PERCOLATE_ABORTED_SCAVENGE was only defined under that CS ifdef. now that it's always defined we can remove ifdefs and make the code marginally more readable

Reply:
I am ok with both ways. I hope compiler can compile out entire section if `IS_CONCURRENT_ENABLED` hardcoded to return false (if OMR_GC_CONCURRENT_SCAVENGER is not defined

Reply:
it definitely should, but even if does not, keeping the code block inactive and performing a runtime check if CS is enabled, once at the start of scavenge cycle is not a big deal

i prefer removing the ifdefs, @kangyining please do so (just for this block)

Reply:
Sure, I'll do that.

Reply:
@amicic I've removed the ifdefs here. Do we want to do another test run on vm farm?